package.json

```diff
@@ -15,15 +16,15 @@
     "node-html-markdown": "^1.3.0",
     "rss-parser": "^3.13.0",
     "toad-scheduler": "^3.0.0",
-    "lru-cache": "^10.0.0"
+    "lru-cache": "^10.0.0",
+    "@types/node": "^20.3.1",
+    "typescript": "^5.1.3"
```

Comment:
Generally speaking, these aren't required for actually running the bot - the build process right now certainly necessitates having them here, considering your addition of the `buildImage` script, but I will probably opt to separate TypeScript compilation from building the docker container in the future, allowing these to be moved back into devDeps.

Totally fine for now though :+1:

Reply:
Right, I couldn't figure out how to improve it further quickly, but that would slim it further.